File: artifacts/codegen/output.js
process.stdout.write("Enter a number: ");
let num = parseInt(prompt("Input:") || "0");
process.stdout.write(`Multiplication Table of ${num}:` + "\n");
let i = 1;
while ((i <= 10)) {
    process.stdout.write(`${num} x ${i} = ${(num * i)}` + "\n");
    i = (i + 1);
}
return 0;

File: artifacts/codegen/output.cpp
#include <iostream>
using namespace std;

int main() {
    int i, num;

    cout << "Enter a number: ";
    cin >> num;
    cout << "Multiplication Table of " << num << ":" << endl;
    i = 1;
    while ((i <= 10)) {
        cout << num << " x " << i << " = " << (num * i) << endl;
        i = (i + 1);
    }
    return 0;
}
